server/PokerTableObjects/Card.js

```diff
@@ -0,0 +1,38 @@
+// this class will most likely be removed from the final design, it was built primarly for testing the
+// intial poker hand rank calculator which had its own UI
+module.exports = class Card {
+    constructor(number, suit) {
+        this.number = number;
+        this.suit = suit;
+    }
+    
+    static getSuit(number){
+        if(number === 1){
+            return 'S';
+        }
+        else if(number === 2){
+            return 'C';
+        }
+        else if(number === 3){
+            return 'H';
+        }
+        else{
+            return 'D';
+        }
+    }
+    
+    static getSuitNumber(suit){
+        if(suit === 'S'){
+            return 1;
+        }
+        else if(suit === 'C'){
+            return 2;
+        }
+        else if(suit === 'H'){
+            return 3;
+        }
+        else{
+            return 4;
+        }
+    }
```

Comment:
Same here.  Would this benefit from being an enum?

Reply:
updated to switch statements